python/ingestion/dataset_utils.py

```diff
@@ -13,7 +13,7 @@ def generate_pct_share_col(df, raw_count_col, pct_share_col, breakdown_col, tota
        total_val: The value representing 'ALL' or 'TOTAL'"""
 
     def calc_pct_share(record, total_value):
-        record[pct_share_col] = round((float(record[raw_count_col]) / float(total)) * 100, 1)
+        record[pct_share_col] = percent_avoid_rouding_to_zero(record[raw_count_col], total)
```

Comment:
is it worth fixing the typo in the function name? `rounding_to`

Reply:
thank you!